locale/zh-cn/docs/guides/event-loop-timers-and-nexttick.md

```diff
@@ -261,30 +262,30 @@ const server = net.createServer(() => {}).listen(8080);
 server.on('listening', () => {});
 ```
 
-只有端口通过时，端口才会立即被绑定。因此，可以立即调用 `'listening'` 回调。问题是 `.on('listening')` 回调将不会被设置的时间。
+只有端口被传递时，端口才会立即被绑定。因此，可以立即调用 `'listening'` 回调。问题是 `.on('listening')` 的回调在那个时间点尚未被设置。
 
-为了绕过此现象，`'listening'` 事件在 `nextTick()` 中排队，以允许脚本运行到完成阶段。这允许用户设置所需的任何事件处理程序。
+为了绕过这个问题，`'listening'` 事件被排在 `nextTick()` 中，以允许脚本运行完成。这让用户设置所想设置的任何事件处理器。
 
 ## `process.nextTick()` 对比 `setImmediate()`
 
-就用户而言我们有两个类似的调用，但它们的名称令人费解。
+就用户而言，我们有两个类似的调用，但它们的名称令人费解。
 
 * `process.nextTick()` 在同一个阶段立即执行。
-* `setImmediate()` 在以下迭代或 ‘tick’ 上触发事件循环。
+* `setImmediate()` 在事件循环的接下来的迭代或'tick'上触发。
```

Comment:
add space between English or special characters and Chinese characters.

Please read the instruction above and self-review this article again.